kaldb/src/main/java/com/slack/kaldb/elasticsearchApi/OpenSearchRequest.java

```diff
@@ -155,6 +156,28 @@ private static List<KaldbSearch.SearchRequest.SearchAggregation> getRecursive(Js
                                       .setField(getFieldName(avg))
                                       .setMissing(SearchResultUtils.toValueProto(getMissing(avg)))
                                       .build());
+                        } else if (aggregationObject.equals(UniqueCountAggBuilder.TYPE)) {
```

Comment:
This is something that I intend to address in a future PR. I don't have all of the possible types of aggregations yet, so I want to make sure that they all need constructed the same way before attempting to extract a common pattern. I have some WIP currently for pipeline aggregators that looks like it will require some further changes to how those are initialized/constructed.